docs/admin/support.rst

```diff
@@ -1,37 +1,37 @@
 Getting support for Weblate
 ===========================
 
-Weblate is a copylefted libre software with community support. Subscribers
-receive priority support at no extra charge. Prepaid help packages are
-available for everyone. You can find more information about current support
+
+Weblate is copylefted libre software with community support.
+Subscribers receive priority support at no extra charge. Prepaid help packages are
+available for everyone. You can find more info about current support
 offerings at <https://weblate.org/support/>.
 
 .. _activate-support:
 
-Activating support
+Integrating support
 ------------------
 
-Since Weblate 3.8 the purchased support packages can be activated in a Weblate
-management interface. The activation will enable peridic registration on
-Weblate servers. This allows you to directly navigate to your Weblate
-installation from subscription management and sends basic telemetry to Weblate
-servers.
 
-.. image:: /images/support.png
+.. versionadded:: 3.8
 
-Data submitted to the server
-----------------------------
+Purchased support packages can optionally be integrated into your Weblate
+subscription management interface, from where you will find a link to it.
```

Comment:
Is this https://weblate.org/user/ and if so, would a link be in order?

Reply:
Yes, that's it. Link would probably make sense here.